### ISSUE TITLE
[Possible Bug]: Serialized model using `TorchModuleWrapper` does not reproduce evaluation metrics

### ISSUE THREAD

**Issue by soumik12345** [keras-team/keras-core#901]
I noticed that when a Keras model using `TorchModuleWrapper` is serialized and evaluated after training, it does not produce the same metrics (accuracy and loss) upon being evaluated. In fact, it seems that the weights are not being serialized properly.

Colab to reproduce issue: https://colab.research.google.com/drive/1IweLgXnyCNY9a2VJ2UkCqvntDurfLobN?usp=sharing

**Comment by fchollet** [comment]
Thanks for the report. Does it work with `save_weights`/`load_weights`?

I think we may have to implement `get_config`/`from_config`/`build_from_config` and possibly even `load_own_variables`.

**Comment by soumik12345** [comment]
Yes, saving the weight worked!

**Comment by fchollet** [comment]
Can you investigate the deserialization process? Basically we'll want to make sure we can recreate the same module, and then we'll want to make sure that `load_own_variables` gets the right variable values and is able to set them to the module params.
